src/views/Profile/index.js

```diff
@@ -0,0 +1,174 @@
+import Grid from 'material-ui/Grid';
+import React, { Component } from 'react';
+import Divider from 'material-ui/Divider/Divider';
+import Card, { CardContent, CardHeader } from 'material-ui/Card';
+import Button from 'material-ui/Button';
+import Typography from 'material-ui/Typography';
+import List from 'material-ui/List/List';
+import ListItem from 'material-ui/List/ListItem';
+
+import user from './../../services/user';
+import { gordonColors } from '../../theme';
+import Activities from './Components/ActivityList';
+import GordonLoader from './../../components/Loader';
+
+export default class Profile extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleExpandClick = this.handleExpandClick.bind(this);
+
+    this.state = {
+      unsername: String,
+      button: String,
+      image: null,
+      loading: true,
+      profile: {},
+      activities: [],
+    };
+  }
+  handleExpandClick() {
+    this.changePrivacy();
+    user.toggleMobilePhonePrivacy();
+  }
+
+  changePrivacy() {
+    if (this.state.button === 'Make Public') {
+      this.setState({ button: 'Make Private' });
+    } else {
+      this.setState({ button: 'Make Public' });
+    }
+  }
+
+  componentWillMount() {
+    const { username } = this.props.match.params.username;
+    this.loadProfile();
+  }
+  async loadProfile() {
+    this.setState({ loading: true });
+    try {
+      const profile = await user.getProfileInfo();
+      this.setState({ loading: false, profile });
+      const [{ def: defaultImage, pref: preferredImage }] = await Promise.all([
+        await user.getImage(),
+      ]);
+      const activities = await user.getMemberships(profile.ID);
+      const image = preferredImage || defaultImage;
+      this.setState({ image, loading: false, activities });
+    } catch (error) {
+      this.setState({ error });
+    }
+    if (this.state.profile.IsMobilePhonePrivate === 0) {
+      this.setState({ button: 'Make Private' });
+    } else {
+      this.setState({ button: 'Make Public' });
+    }
+  }
+  render() {
+    console.log(this.state.profile);
+
+    const style = {
+      width: '100%',
+    };
+    const button = {
+      background: gordonColors.primary.cyan,
+      color: 'white',
+    };
+
+    let activityList;
+    if (!this.state.activities) {
+      activityList = <GordonLoader />;
+    } else {
+      activityList = this.state.activities.map(activity => (
+        <Activities Activity={activity} key={activity.MembershipID} />
+      ));
+    }
+
+    let content = (
+      <Grid container>
```

Comment:
I think this `<Grid container>` should have `justify="center"` ![image](https://user-images.githubusercontent.com/25458858/36102200-9a9dcc50-0fd9-11e8-8d12-eea5aaada090.png)
And I'm not sure if you can do anything about the text wrapping for sm